src/Vision/VNUtils.cs

```diff
@@ -61,6 +61,13 @@ public static CGPoint GetImagePoint (Vector2 faceLandmarkPoint, CGRect faceBound
 
 			return result;
 		}
+
+		[TV (13,0), Mac (10,15), iOS (13,0)]
+		[DllImport (Constants.VisionLibrary, EntryPoint = "VNElementTypeSize")]
+		static extern nuint _GetElementTypeSize (nuint elementType);
```

Comment:
why use `EntryPoint` ? it's an internal symbol so `VNElementTypeSize` should be fine right ?

Reply:
Yeah, I have no answer but I just like the other name heh, anyways I'll change it :)